invenio_rdm_records/services/schemas/access.py

```diff
@@ -76,6 +76,9 @@ class AccessSchema(Schema):
     # TODO re-enable when the grants are ready
     # grants = fields.List(fields.Nested(Grant))
 
+    # NOTE: this field is only present to avoid errors for unknown fields
+    grant_tokens = fields.List(fields.String, dump_only=True)
```

Comment:
Minor: I'm puzzled why this one is here? The tokens should only exist in the index, and not make their way out in the REST API.

Reply:
Removing that doesn't break the tests now, so it's removed.